apigen-deps/src/main/java/com/distelli/graphql/ResolverDataFetcher.java

```diff
@@ -1,23 +1,36 @@
 package com.distelli.graphql;
 
-import graphql.schema.DataFetcher;
-import graphql.schema.DataFetchingEnvironment;
-import graphql.execution.batched.Batched;
-import graphql.execution.batched.BatchedDataFetcher;
-import java.util.List;
+import java.lang.reflect.Method;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Iterator;
-import java.lang.reflect.Method;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import graphql.execution.batched.Batched;
+import graphql.execution.batched.BatchedDataFetcher;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.DataFetchingEnvironmentImpl;
+
+import static com.google.common.collect.Lists.newArrayList;
```

Comment:
I'm pretty sure that we are only using Guava indirectly through Guice. Given that we have users of this library that do not use Guice but instead use Spring (and thus do not have a dependency on Guice), I think using this library will break those users.